### PR TITLE
resolve path setting problem in macOS

### PR DESCRIPTION
**Pull request by rookiewangsl**
The path for mh_style in mac is not in the default path of matlab, therefore add an path setting in line 15
Remember to substitute your own username for rookie in line 15;
Because I have tried ~ for convenience but failed

### CODE REVIEW

File: MISS_HIT.m [hunk 1]
```diff
@@ -11,6 +11,8 @@ function mh_style(varargin)
             if ismac
                 suppress_output = '1> /dev/null 2> /dev/null';
                 set_environment = 'export PYTHONIOENCODING=UTF-8 && ';
```
Comment (Sec-ant) [inline MISS_HIT.m:13]: Maybe this should be replaced by MATLAB `setenv` function to be consistent with the following path setting statement.

File: MISS_HIT.m [hunk 2]
```diff
@@ -11,6 +11,8 @@ function mh_style(varargin)
...
+                % add path(substitute your own user name for rookie)
+                setenv('PATH', getenv('PATH')+":/Users/rookie/.local/bin/")
```
Comment (Sec-ant) [inline MISS_HIT.m]: Is there any chance that the username can be acquired dynamically? What's the output of `system("id -un")`.

**Review by Sec-ant — COMMENTED** [review]
Related discussions on this issue in MATLAB Answers: https://www.mathworks.com/matlabcentral/answers/415984-why-can-t-i-call-unix-commands-from-matlab-on-mac-os

**Comment by Sec-ant** [comment]
I'll recheck this problem on linux and let's see.

File: MISS_HIT.m (hunk 2 above)
Comment (rookiewangsl) [reply MISS_HIT.m]: done

Comment (rookiewangsl) [reply MISS_HIT.m]: how to cite the revised code like u in pull request conversation

Comment (Sec-ant) [reply MISS_HIT.m]: Click one of your commit and a "+" will pop up when you move your mouse to a certain line number and you can add comments on it. 
image

File: MISS_HIT.m [hunk 3]
```diff
@@ -24,6 +26,7 @@ function mh_style(varargin)
 
             % default options
             default_args = {['--fix ', suppress_output]};
+%             default_args = {['--fix ']};
```
Comment (Sec-ant) [inline MISS_HIT.m]: This line should be removed

**Review by Sec-ant — COMMENTED** [review]
This file itself (MISS_HIT.m) should be formatted by `mh_style`

**Pull request merged**